getlantern.org/static/js/download.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function(){
+
+  var ANDROID_LINK = "lantern-installer-beta.apk";
+
+  var set_download_link = function() {
+    var os = platform.os.architecture + platform.os.family;
+    var os_links = [
+      {regexp: "[2-6]{2}[Ww]indows", link: "lantern-installer-beta.exe"},
+      {regexp: "[2-6]{2}OS X", link: "lantern-installer-beta.dmg"},
+      // for Android devices with large screen
```

Comment:
Note that the download link for Android is move above Linux, as the os can be `32Android` sometimes.